### PR TITLE
Remove experimental feature labels and flags for OS Assistant

### LINKED ISSUE #6713: [DOC] OpenSearch Assistant going GA
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 The assistant is going GA in 2.13, so we need to remove the experimental flag.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
https://github.com/opensearch-project/dashboards-assistant/issues/18

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Closes #6713 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by lezzago — APPROVED** [review]

**Review by hdhalter — APPROVED** [review]
LGTM

File: _observing-your-data/event-analytics.md [hunk 1]
```diff
@@ -39,21 +39,22 @@ To simplify query building, the **OpenSearch Assistant** toolkit offers an assis
...
 #### Enabling Query Assistant
 
-To enable **Query Assistant** in OpenSearch Dashboards, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
+By default, **Query Assistant** is enabled in OpenSearch Dashboards. To configure Query Assistant, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
 
-```
-observability.query_assist.enabled: true
+```yaml
 observability.query_assist.ppl_agent_name: "PPL agent"
 ```
```
Comment (lezzago) [inline _observing-your-data/event-analytics.md:46]: This part is no longer valid.

File: _observing-your-data/event-analytics.md [hunk 2]
```diff
@@ -39,21 +39,22 @@ To simplify query building, the **OpenSearch Assistant** toolkit offers an assis
 
 #### Enabling Query Assistant
 
-To enable **Query Assistant** in OpenSearch Dashboards, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
+By default, **Query Assistant** is enabled in OpenSearch Dashboards. To configure Query Assistant, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
 
-```
-observability.query_assist.enabled: true
```
Comment (lezzago) [inline _observing-your-data/event-analytics.md:48]: We can keep this removed part and say if they want to disable it, they can set this setting to false.

File: _observing-your-data/event-analytics.md (hunk 1 above)
Comment (kolchfa-aws) [reply _observing-your-data/event-analytics.md:46]: Removed.

File: _observing-your-data/event-analytics.md (hunk 2 above)
Comment (kolchfa-aws) [reply _observing-your-data/event-analytics.md:48]: I added a sentence about that on line 50.

**Pull request merged**